lib/src/components/global_header/global_header.dart

```diff
@@ -0,0 +1,162 @@
+
+  @override
+  State<ZetaGlobalHeader> createState() => _GlobalHeaderState();
+
+  @override
+  void debugFillProperties(DiagnosticPropertiesBuilder properties) {
+    super.debugFillProperties(properties);
+    properties.add(StringProperty('title', title));
+  }
+}
+
+extension on DeviceType {
+  /// Render buttons along the top menu half
+  bool get isLarge {
+    return this == DeviceType.desktopL || this == DeviceType.desktopXL;
+  }
+
+  /// Render
+  bool get isSmall {
+    return this == DeviceType.mobileLandscape || this == DeviceType.mobilePortrait;
+  }
+}
+
+class _GlobalHeaderState extends State<ZetaGlobalHeader> {
+  int _selectedIndex = -1;
+
+  @override
+  void initState() {
+    super.initState();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    final colors = Zeta.of(context).colors;
+    return LayoutBuilder(
+      builder: (context, constraints) {
+        final deviceType = constraints.deviceType;
+
+        return Container(
+          padding: const EdgeInsets.symmetric(
+            vertical: ZetaSpacing.s,
+            horizontal: ZetaSpacing.b,
+          ),
+          decoration: BoxDecoration(color: colors.surfacePrimary),
+          child: Column(
+            children: [
+              Row(
+                mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                // Top Section
+                children: [
+                  Row(
+                    children: [
+                      Text(
+                        widget.title,
+                        style: TextStyle(
+                          color: colors.textDefault,
+                          fontWeight: FontWeight.w500,
+                        ),
+                      ),
+                      const SizedBox.square(
+                        dimension: ZetaSpacing.s,
+                      ),
+                      if (deviceType.isLarge && widget.tabItems.length > 5)
+
+                        /// If using large screen, render some tabItems in to section
+                        ...renderedChildren(widget.tabItems).sublist(0, 4),
+                    ],
+                  ),
+
+                  /// If screen is not small, render search bar on the top
+                  if (!deviceType.isSmall)
+                    const Expanded(
+                      child: ZetaSearchBar(),
+                    ),
+                  Row(
+                    children: [
+                      ...widget.utilityButtons,
+                      const ZetaAvatar(
+                        initials: 'PS',
+                        size: ZetaAvatarSize.s,
+                      ),
+                    ].gap(ZetaSpacing.s),
+                  ),
+                ].gap(ZetaSpacing.s),
+              ),
+              const SizedBox(
+                height: ZetaSpacing.x2,
+              ),
+              Row(
+                children: [
+                  if (deviceType.isSmall) const Expanded(child: ZetaSearchBar()),
```

Comment:
I think the medium screen layout is wrong - this should have the searchbar on the top row, and the buttons below but currently that is reversed:
<img width="755" alt="image" src="https://github.com/zebrafed/zeta_flutter/assets/23633655/6c7af4ea-9f85-4352-80f3-d2dea127eeaa">

Reply:
That screen is the small one. I asked phil on figma how we should respond to the situation on the small screen where we have those tab items. no response as of yet

Reply:
I changed it so now landscape mobiles have a medium display